libindy/build_scripts/android/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:16.04
```

Comment:
I believe we need to rename it to something more obvious like android-build-env.dockerfile